NIST_V2_election_results_reporting.xsd

```diff
@@ -491,11 +499,27 @@
     </xsd:sequence>
     <xsd:attribute name="Label" type="xsd:string"/>
   </xsd:complexType>
+  <xsd:complexType name="OrderedContent" abstract="true">
```

Comment:
What is OrderedContent? It doesn't seem to have any elements. If it's supposed to have contests, headers, and ballot selections, why not include them here instead of below?

Reply:
`OrderedContent` is the base class for `OrderedContest` or `OrderedHeader`. They share no attributes, thus the parent class is devoid of any content.